src/providers/v2/subgraph-provider.ts

```diff
@@ -39,8 +39,6 @@ type RawV2SubgraphPool = {
 const SUBGRAPH_URL_BY_CHAIN: { [chainId in ChainId]?: string } = {
   [ChainId.MAINNET]:
     'https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2',
-  [ChainId.RINKEBY]:
-    'https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-v2-rinkeby',
```

Comment:
wow didn't realize we have a provider url based on personal employee's account..

Reply:
no action item for now I think.